Cargo.toml

```diff
@@ -13,3 +13,21 @@ rust-version = "1.75.0"
 license = "Apache-2.0"
 homepage = "https://github.com/koushiro/fastmetrics"
 repository = "https://github.com/koushiro/fastmetrics"
+
+[workspace.dependencies]
```

Comment:
Can you revert the dependency changes in this repo? 
The crates in this repo do not have duplicate dependencies.

Reply:
done